README.md

```diff
@@ -14,7 +14,13 @@ The static marketing site for login.gov
 
 ## Development
 
-This is a Jekyll-built static site. To start serve the site locally in development:
+This is a Jekyll-built static site. To install dependencies:
+
+```
+bundle install
```

Comment:
Would you mind updating the PR in-place to do it, or editing after merge? It's not clear to me whether that's one line or two, and I didn't use those commands in my own work and am not in a place to test them right now.